custom/icds/management/commands/migrate_pse_attendance.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import print_function
+
+from django.core.management.base import BaseCommand
+
+from django.db import connections
+
+
+new_table = 'config_report_icds-cas_static-child_cases_monthly_v2_198ccc06'
+old_table = 'config_report_icds-cas_static-child_cases_monthly_tabl_551fd064'
+
+migration_query = """
+UPDATE {new_table} B
+SET
+B.pse_daily_attendance = A.pse_days_attended,
+B.pse_daily_attendance_male = CASE WHEN B.sex = 'M' THEN A.pse_days_attended ELSE NULL END,
+B.pse_daily_attendance_female = CASE WHEN B.sex = 'F' THEN A.pse_days_attended ELSE NULL END
+FROM {old_table} A
+WHERE A.doc_id = B.doc_id and A.month_start = B.month_start
```

Comment:
doc_id, month_start defines a unique record. i believe doc_id is indexed and potentially an explicit primary key and there are exactly 7 month_starts for each doc_id. I will want to coordinate this process with emord as he has been managing the ongoing rebuild of these datasources and want to make sure its alright to stop those processes but it sounds like we are good to go